build.py

```diff
@@ -68,6 +71,15 @@
                 '-I'+Configuration.current.variables["LIBDISPATCH_BUILD_DIR"]+'/tests'  # for include of dispatch/private.h in CF
         ])
 
+if Configuration.current.build_mode == Configuration.Release:
+        foundation.LDFLAGS += ' -lswiftSwiftOnoneSupport '
+        # In the future, remove the LDFLAGS line above and enable
+        # the optimizations. But currently, it results in a segmentation
+        # fault on one of the tests.
```

Comment:
if this causes a segfault in one of the tests and it is currently working that probably means there is an expectation failure that is not met in your library:

it is worth noting the lines below are incorrect and should not be modified at that point:

from lib/script.py:111

```python
        if Configuration.current.build_mode == Configuration.Debug:
            swift_flags += "-g -Onone "
        elif Configuration.current.build_mode == Configuration.Release:
            swift_flags += "-O "

```

that modification runs just fine without your changes; can you get me the crash point by attaching lldb?

```sh
cd swift-corelibs-foundation
ninja test
LD_LIBRARY_PATH=../build/Ninja-ReleaseAssert/foundation-linux-x86_64/Foundation/:/home/phausler/Public/build/Ninja-ReleaseAssert/xctest-linux-x86_64 ../build/Ninja-ReleaseAssert/lldb-linux-x86_64/bin/lldb ../build/Ninja-ReleaseAssert/foundation-linux-x86_64/TestFoundation/TestFoundation
```

presuming you built as release mode in the standard layouts